src/main/resources/assets/create/lang/es_es.json

```diff
@@ -30,13 +30,13 @@
 	"block.create.black_sail": "Vela negra",
 	"block.create.black_seat": "Asiento negro",
 	"block.create.black_valve_handle": "Asa de válvula negra",
-	"block.create.blaze_burner": "Quemador de Blaze",
+	"block.create.blaze_burner": "Blaze Burner",
```

Comment:
Why in English :s

Reply:
That's what I told you... your last changes weren't included because i made changes at the same time as you...